src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -0,0 +1,187 @@
+    };
+
+type WorkspaceListPageOnyxProps = {
+    /** The list of this user's policies */
+    policies: OnyxCollection<PolicyType>;
+
+    /** Bank account attached to free plan */
+    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
+
+    /** A collection of objects for all policies which key policy member objects by accountIDs */
+    allPolicyMembers: OnyxCollection<PolicyMembers>;
+};
+
+type WorkspaceListPageProps = WithPolicyAndFullscreenLoadingProps & WorkspaceListPageOnyxProps;
+
+const workspaceFeatures = [
+    {
+        icon: Illustrations.MoneyReceipts,
+        translationKey: 'workspace.emptyWorkspace.features.trackAndCollect',
+    },
+    {
+        icon: Illustrations.CreditCardsNew,
+        translationKey: 'workspace.emptyWorkspace.features.companyCards',
+    },
+    {
+        icon: Illustrations.MoneyWings,
+        translationKey: 'workspace.emptyWorkspace.features.reimbursements',
+    },
+];
+
+/**
+ * Dismisses the errors on one item
+ */
+function dismissWorkspaceError(policyID: string, pendingAction: OnyxCommon.PendingAction) {
+    if (pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE) {
+        Policy.clearDeleteWorkspaceError(policyID);
+        return;
+    }
+
+    if (pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD) {
+        Policy.removeWorkspace(policyID);
+        return;
+    }
+    throw new Error('Not implemented');
+}
+
+function WorkspacesListPage({policies, allPolicyMembers, reimbursementAccount}: WorkspaceListPageProps) {
+    const theme = useTheme();
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const {isOffline} = useNetwork();
+
+    /**
+     * Gets the menu item for each workspace
+     */
+    const getMenuItem = (item: WorkspaceItem, index: number) => (
+        <OfflineWithFeedback
+            key={`${item.title}_${index}`}
+            pendingAction={item.pendingAction}
+            errorRowStyles={styles.ph5}
+            onClose={item.dismissError}
+            errors={item.errors}
+        >
+            <MenuItem
+                title={item.title}
+                icon={item.icon}
+                iconType={CONST.ICON_TYPE_WORKSPACE}
+                onPress={item.action}
+                iconFill={theme.textLight}
+                shouldShowRightIcon
+                fallbackIcon={Expensicons.FallbackWorkspaceAvatar}
+                brickRoadIndicator={item.brickRoadIndicator}
+                disabled={item.disabled}
+            />
+        </OfflineWithFeedback>
+    );
+
+    /**
+     * Add free policies (workspaces) to the list of menu items and returns the list of menu items
+     */
+    const workspaces: WorkspaceItem[] = useMemo(() => {
+        const reimbursementAccountBrickRoadIndicator = reimbursementAccount?.errors ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined;
+        if (isEmptyObject(policies)) {
+            return [];
+        }
+        return Object.values(policies ?? {})
```

Comment:
```suggestion
        return Object.values(policies)
```

Is the nullish check needed here?

Reply:
I think we need it because if the policies are undefined or null, Object.values(policies) will through an error

Reply:
I'm not seeing any error. I think we already covered that case with the isEmptyObject check

Reply:
You are correct. Removed it